libcodechecker/util.py

```diff
@@ -486,7 +486,7 @@ def escape_like(string, escape_char='*'):
                  .replace('_', escape_char + '_')
 
 
-def get_line(file_name, line_no, errors='replace'):
+def get_line(file_name, line_no, errors='ignore'):
```

Comment:
Please change it in the following place too:
https://github.com/Ericsson/codechecker/blob/e83962f194932d96b3d752a544c1cc410a3b9607/libcodechecker/cmd/cmd_line_client.py#L704-L705